docs/DeveloperGuide.md

```diff
@@ -641,3 +641,35 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+
+### Class `Record` JSON Storage Format
+
+**Format:** A `Record` object is represented in JSON as follows:
+
+```json
+{
+    "patient": {
+        "name": "John Doe",
+        "phone": "98765432",
+        "email": "johnd@example.com",
+        "gender": "MALE",
+        "icNumber": "S2840182A",
+        "birthday": "02/01/1998",
+        "address": "311, Clementi Ave 2, #02-25",
+        "tags": ["friend", "owesMoney"]
+    },
+    "initialObservations": "Patient complains of a persistent cough for 2 weeks.",
+    "diagnosis": "Common cold, aggravated due to not taking enough rest.",
+    "treatmentPlan": "Rest, Hydration, and prescribed cough syrup."
+}
+```
+
+#### Explanation:
```

Comment:
I like how you added the explanation about how you integrated Record into Storage!